include/common_components.php

```diff
@@ -1,24 +1,384 @@
+         
+            <label for='uphone'><b>Phone</b></label>
+            <input type='tel' name='uphone' placeholder='999-999-9999' pattern='[0-9]{3}-[0-9]{3}-[0-9]{4}' required>
+
+            <label for='urating'><b>ELO Rating</b></label>
+            <input type='number' min='0' max='3000'placeholder='ELO' name='urating' required>       
+          </div>
+      
+          <div class='container' style='background-color:#f1f1f1'>
+              <button type='submit' name='AddStudentSubmitted'>Add</button>
+              <button type='button' onclick= document.getElementById('id01').style.display='none' class='cancelbtn'>Cancel</button>
+          </div>
+        </form>
+      </div>
+    
+      <script>
+        // Get the modal
+        var modal = document.getElementById('id01');
+        
+        // When the user clicks anywhere outside of the modal, close it
+        window.onclick = function(event) {
+            if (event.target == modal) {
+                modal.style.display = 'none';
+            }
+        }
+      </script>   
+";
+}
+
+// --------------------------------------------------------------------------
+
+function createAddCreditsForm(){
+    echo"          
+        <div class='form-popup' id='addCreditsForm'>
+            <form action='' class='form-container'>                
+                <button type='submit' class='btn' name='AddCredditsSubmitted'>Add</button>
+                <button type='button' class='btn cancel' onclick='closeCreditForm()'>Close</button>
+
+                <label for='camount'><b>Credits Amount</b></label>
+                <input type='number' min='0' max='100' placeholder='0 - 100' name='camount' required>
+
+                <label for='id'><b></b></label>
+                <input type='hidden' name='id' value=$_REQUEST[id]>        
+            </form>
+        </div>
+        
+        <script>
+            function openCreditForm() {
+                document.getElementById('addCreditsForm').style.display = 'block';
+            }
+            
+            function closeCreditForm() {
+                document.getElementById('addCreditsForm').style.display = 'none';
+            }
+        </script>      
+    ";
+}
+
+// --------------------------------------------------------------------------
+
+function createAddClassForm(){
+    echo"          
+        <div class='form-popup' id='addClassForm'>
+            <form action='' class='form-container'>
+                <button type='button' class='btn cancel' onclick='closeClassForm()'>Close</button>
+                <button type='submit' class='btn' name='AddClassesSubmitted'>Book</button>
+
+                <label for='ctype'>Class Type</label>
+                <select id='dropdown' name='ctype'>
+                    <option value='Online'>Online</option>
+                    <option value='In Person'>In Person</option>                  
+                </select>   
```

Comment:
This is so true! I'm giving In-person classes as an option because he does in person classes but as of right now all his classes are online. I will talk to Yuni about this